packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAPPaymentQueueDelegate.h

```diff
@@ -9,13 +9,14 @@
 #endif
 #import <Foundation/Foundation.h>
 #import <StoreKit/StoreKit.h>
+#import "MethodChannelProtocol.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 API_AVAILABLE(ios(13))
 API_UNAVAILABLE(tvos, macos, watchos)
 @interface FIAPPaymentQueueDelegate : NSObject <SKPaymentQueueDelegate>
-- (id)initWithMethodChannel:(FlutterMethodChannel *)methodChannel;
+- (id)initWithMethodChannel:(id<MethodChannel>)methodChannel;
```

Comment:
Nit: all objc types need to be prefixed

Reply:
`FlutterMethodChannel` naming itself isn't following the convention, but new types should follow it. How about `FLTMethodChannelProtocol`.

Reply:
should i prefix the other protocols too

Reply:
i just did it anyways